### ISSUE TITLE
Adding FormData to WebWorker

### ISSUE THREAD

**Issue by dmikis** [Microsoft/TSJS-lib-generator#196]
I'm trying to add `FormData` interface to `webworker.d.ts` (https://github.com/Microsoft/TypeScript/issues/14085). IIUC, I need to add to `inputfiles/knownWorkerInterfaces.json`. It generates:

```ts
interface FormData {
    append(name: any, value: any, blobName?: string): void;
}

declare var FormData: {
    prototype: FormData;
    new (form?: HTMLFormElement): FormData;
}
```

The problem is the `FormData`'s constructor. Is there a way to add different constructors for DOM and Workers?

**Comment by SaschaNaz** [comment]
This is fixed by #231.

**Comment by dmikis** [comment]
Cool!
